configuration/concepts/diagnoses-covid.csv

```diff
@@ -0,0 +1,27 @@
+uuid,Void/Retire,Fully specified name:es,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Answers,Mappings|SAME-AS|PIH|Name,Mappings|SAME-AS|PIH|Number,Mappings|SAME-AS|CIEL,Mappings|SAME-AS|AMPATH,Mappings|SAME-AS|ICD-10-WHO,Mappings|NARROWER-THAN|ICD-10-WHO,Mappings|NARROWER-THAN|ICD-10-WHO 2nd,Mappings|NARROWER-THAN|ICD-11-WHO,Mappings|SAME-AS|org.openmrs.module.emrapi,Mappings|SAME-AS|PIH Malawi,Mappings|SAME-AS|SNOMED CT,Mappings|NARROWER-THAN|SNOMED CT,Mappings|SAME-AS|SNOMED UK,_order:100
```

Comment:
In the existing system, the Mexico diagnoses were comprised of a "set of sets", so I created a top-level file for each set... in my opinion, this will make manintenane easier.

In each file, I put the set at the top (which we can do since we are now configuring the set members in a separate file).

With a single diagnosis set per file, creating the necessary "concept set" file can be done by the just copying from this file into the set file whatever column you are doing a lookup on.